aea/protocols/dialogue/base.py

```diff
@@ -1077,7 +1077,12 @@ def __init__(self, dialogues: "Dialogues") -> None:
         self._incomplete_to_complete_dialogue_labels = (
             {}
         )  # type: Dict[DialogueLabel, Optional[DialogueLabel]]
-        self._dialogues = dialogues
+        self._dialogues_info = {
+            "is_keep_dialogues_in_terminal_state": dialogues.is_keep_dialogues_in_terminal_state,
+            "class_name": dialogues.__class__.__name__,
+            "dialogue_class": dialogues.dialogue_class,
+            "message_class": dialogues.message_class,
+        }
```

Comment:
I'm just pushing this as I had forgotten to do it yesterday when I rolled back these changes from the dialogue PR. There will be failing tests. I still need to look into the issue once I finish my other PRs.